src/resultsdisassemblypage.cpp

```diff
@@ -187,6 +142,9 @@ void ResultsDisassemblyPage::setSymbol(const Data::Symbol& symbol)
 void ResultsDisassemblyPage::setCostsMap(const Data::CallerCalleeResults& callerCalleeResults)
 {
     m_callerCalleeResults = callerCalleeResults;
+    m_model->setResults(m_callerCalleeResults);
+    const auto numTypes = m_callerCalleeResults.selfCosts.numTypes();
+    emit m_model->headerDataChanged(Qt::Horizontal, 0 , numTypes + 1);
```

Comment:
remove, the model is responsible for this, and it should begin/endReset when calling setResults internally, so this shouldn't be needed